cli/wallet/multisig.go

```diff
@@ -18,7 +18,7 @@ func newMultisigCommands() []cli.Command {
 		outFlag,
 		inFlag,
 		cli.StringFlag{
-			Name:  "addr",
+			Name:  "address, a",
```

Comment:
Usage text should also be updated (line 30).

Reply:
Fixed.